haphilipsjs/__init__.py

```diff
@@ -114,15 +114,12 @@ def setChannel(self, id):
             self.channel_id = id
 
     def getChannelLists(self):
-        if self.api_version >= '6':
-            r = self._getReq('channeldb/tv')
-            if r:
-                # could be alltv and allsat
-                return [l['id'] for l in r.get('channelLists', [])]
-            else:
-                return []
+        r = self._getReq('channeldb/tv')
```

Comment:
Wouldn't it be safer to replace the `6` with a `5`? Otherwise you'd be breaking it for everyone with API < 5 in case that's not supported in those versions.

Reply:
getChannellists is only used in getSources with API > 5. Depends on the philosophy. If you want it safe you are right.

Reply:
> Depends on the philosophy. If you want it safe you are right.

Of course. Not breaking the integration for users with older models is the primary purpose of checking the API version. And that's not acceptable in my opinion.

Reply:
This getChannelist method was never used in the context of API 1/<5 (in Home Assistent).You are correct. I can write the solution with the API Check.

Reply:
I had a deeper look at it now. You're right, that this won't be called for lower API versions. So I guess this won't break anything.